lms/static/scripts/frontend_apps/components/AppRoot.tsx

```diff
@@ -41,10 +41,10 @@ export default function AppRoot({ initialConfig, services }: AppRootProps) {
               <FilePickerApp />
             </DataLoader>
           </Route>
-          <Route path="/dashboard/orgs/:organizationPublicId" nest>
+          <Route path="/dashboard/orgs/:organizationPublicId/" nest>
             <DashboardApp />
           </Route>
-          <Route path="/dashboard" nest>
+          <Route path="/dashboard/" nest>
```

Comment:
I'm not completely sure if these changes are needed. It actually works without them 🤔 

Considering the nested routes all define the leading bar, it probably makes more sense to not define these, as we end up with routes like `"/dashboard/" + "/assignments/{id}/"`, instead of `"/dashboard" + "/assignment/{id}/"`, but I guess the router internally removes consecutive duplicated slashes and that's why it works.

Reply:
I have dropped these.